coda-prototype/src/ui/tableUI.js

```diff
@@ -1326,59 +1326,52 @@ var messageViewerManager = {
         let nextScheme = eventObj.firstUncodedScheme(messageViewerManager.codeSchemeOrder);
         if (nextScheme.length > 0) {
             // pass the columns until reaching this scheme
-            setTimeout(() => {
-                let immutableSchemeOrder = messageViewerManager.codeSchemeOrder.slice();
-                for (let schemeKey of immutableSchemeOrder) {
-                    if (schemeKey === nextScheme) {
-                        break;
-                    } else {
-                        messageViewerManager.changeActiveScheme();
-                        messageViewerManager.resizeViewport();
-                    }
+            let immutableSchemeOrder = messageViewerManager.codeSchemeOrder.slice();
+            for (let schemeKey of immutableSchemeOrder) {
+                if (schemeKey === nextScheme) {
+                    break;
+                } else {
```

Comment:
This doesn't need to be in an else block, it's better to extract it one level out